build.gradle

```diff
@@ -53,38 +57,34 @@ sourceSets {
 }
 
 dependencies {
-    implementation deps.kotlin.jdk
+    api deps.kotlin.jdk
+
+    api deps.logs.logback
+    api deps.logs.logbackJson
+    api deps.logs.coroutinesSlf4j
 
-    implementation deps.logs.logback
-    implementation deps.logs.logbackJson
-    implementation deps.logs.coroutinesSlf4j
+    api deps.ktor.serverNetty
+    api deps.ktor.serialization
+    api deps.ktor.auth
 
-    implementation deps.ktor.serverNetty
-    implementation deps.ktor.serialization
-    implementation deps.ktor.auth
+    api deps.koin.core
+    api deps.koin.coreExt
+    api deps.koin.ktor
 
-    implementation deps.koin.core
-    implementation deps.koin.coreExt
-    implementation deps.koin.ktor
+    api deps.valiktor
 
-    implementation deps.valiktor
-    // Database stuff
-    implementation deps.postgresql
-    implementation deps.hikari
-    implementation deps.exposed.core
-    implementation deps.exposed.dao
-    implementation deps.exposed.jdbc
-    implementation deps.flyaway
+    // Database
+    api deps.exposed.core
+    api deps.exposed.dao
+    api deps.exposed.jdbc
 
+    // Tests
     testImplementation deps.ktor.test
```

Comment:
**api** imports dependencies on the `ktor-pet-clinic` but **testApi** is not working to import test dependencies. Do you know if it is another syntax?

Reply:
Do we need to have a separate bootstrap project for tests that use the **api** annotation like [spring-boot-starter-test](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-starters/spring-boot-starter-test/build.gradle) and then import it with **testImplementation**?

Reply:
I don't known exactly what's the use case for `testApi` but normally we always use `testImplementation` for testing. Don't worry much about this, i'll take care of this on my refactor branch.